ui/pages/remove-snap-account/snap-account-card.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import {
+  getInternalAccounts,
+  getMetaMaskAccountsOrdered,
+} from '../../selectors';
+import { BlockSize, BorderRadius } from '../../helpers/constants/design-system';
+import { Box } from '../../components/component-library';
+import { AccountListItem } from '../../components/multichain';
+import { mergeAccounts } from '../../components/multichain/account-list-menu/account-list-menu';
+
+// Wrapper component of AccountListItem with proper styling and auto populating information for the selected account
+export const SnapAccountCard = ({
+  address,
+  remove,
+}: {
+  address: string;
+  remove?: boolean;
+}) => {
+  const accounts = useSelector(getMetaMaskAccountsOrdered);
+  const internalAccounts = useSelector(getInternalAccounts);
+  // We should stop using mergeAccounts and write a new selector instead
```

Comment:
I understand that there could have been not enough time to implement a new selector but I don't think this comment as is helps anybody.

Reply:
It was meant as a TODO for future iterations. I am gonna forget about this after this PR so I would prefer leaving it, but I am open to remove it if you feel strongly that its useless

Reply:
How about making it explicit todo then
```suggestion
  // TODO(FrederikBolding): We should stop using mergeAccounts and write a new selector instead
```